install/kubernetes/templates/namespace.yaml

```diff
@@ -1,4 +1,4 @@
 apiVersion: v1
 kind: Namespace
 metadata:
- name: {{ .Release.Namespace }}
```

Comment:
can we instead make this conditional ?
While using `helm template` we will render it because certain condition is met.
tiller will not render it.

Reply:
I didn't find a way to tell whether it is running helm template, helm install or helm upgrade.